### PR TITLE
Fix typo in Contributing / issues.md

### PR DESCRIPTION
**Pull request by chalin**
- Followup to #5266

### CODE REVIEW

File: content/en/docs/contributing/issues.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ One of the best ways to help make OTel docs better is to fix an existing issue.
    need for clarification by posting comments over the issue.
 5. Share your intention to work on the issue by adding add a comment to this
    effect.
-6. Work on fixing the issue. Let maintainers now if you run into any problems.
+6. Work on fixing the issue. Let maintainers know if you run into any problems.
```
Comment (chalin) [inline content/en/docs/contributing/issues.md:47]: @open-telemetry/docs-approvers - regarding steps 5 & 6: should we explicitly say that contributors don't have to wait for our thumbs up, or should they wait for at least a thumbs up from a docs approver?

Comment (theletterf) [reply content/en/docs/contributing/issues.md:47]: I'd encourage them not to wait.

Comment (chalin) [reply content/en/docs/contributing/issues.md:47]: That works for me. In that case, I'd leave the step prose as is for now (by not stating that contributors have to wait, they have to liberty of moving on from steps 5 to step 6 directly -- we can always make it explicit later if the need arises).

Comment (chalin) [reply content/en/docs/contributing/issues.md:47]: So, shall we merge this typo fix as is?

**Review by theletterf — APPROVED** [review]

**Pull request merged**